packer/setup.sh

```diff
@@ -63,8 +63,11 @@ User=pachyderm
 WorkingDirectory=/home/pachyderm
 ExecStart=/home/pachyderm/scripts/start.sh
 Environment=PYTHONUNBUFFERED=1
-Restart=always
+Restart=on-failure
```

Comment:
I think we want this to restart even if it exits with status 0, no? Not that it should ever exit with status 0, but leaving `Restart=always` seems like it would minimize the consequences of such a bug?